src/routes/news/components/List.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+import React from 'react'
+import { FlatList } from 'react-native'
+import styled from 'styled-components/native'
+import {
+  LocalNewsModel,
+  TunewsModel
+} from '@integreat-app/integreat-api-client'
+
+const Loader = styled.ActivityIndicator`
+  margin-top: 7px;
+`
+const keyExtractor = (item, index) => `${index}`
+
+interface ListPropTypes {
+  items: Array<LocalNewsModel | TunewsModel>,
+  renderItem: ({
+    cityCode: string,
+    language: string,
+    item: LocalNewsModel
```

Comment:
Why only `LocalNewsModel` here?

Reply:
It should be both models. it was missed while refactoring. Will add it